src/elements/content-sidebar/ContentSidebar.js

```diff
@@ -67,6 +67,7 @@ type Props = {
     hasAdditionalTabs: boolean,
     hasMetadata: boolean,
     hasNav: boolean,
+    hasPanelSelectionPreservation: boolean,
```

Comment:
Ok, If I'm following correctly, this is to keep the last selected metadata option the user chose right? So if the user opened a file and then closes it, later they chose another file the selection is preserved?

Reply:
Not really. This is not strictly connected to metadata. The goal of the change is for the Sidebar to remember the last opened panel. Let's say user opens sidebar and sees the `activity` panel in sidebar (first one that this user has access to). If user then opens `details` panel it will be stored on local storage, so when the user opens preview next time, `details` will be the active panel.

Reply:
this prop needs to be moved to `features` prop

Reply:
Done